intro.md

```diff
@@ -0,0 +1,17 @@
+Welcome to Win32 App Isolation Introduction Page
+
+Target Applications
+This feature is intended to increase the security of Win32 and Centennial apps.
+
+Minimum requirements:
+•	Check that Windows version is 25357 or greater
+•	One will need to download both MPT v1.2023.517.0 and ACP (Releases · microsoft/win32-app-isolation (github.com))
+•	In addition, it is necessary to have Windows Performance Recorder, get it from here
+
+Creating a Silo App:
+What you will see below is a summary of the steps to follow to turn a Win32 App into a Silos App. For more detailed information like the manifest changes, please visit the following link: win32-app-isolation/docs/packaging/msix-packaging-tool.md at main · microsoft/win32-app-isolation (github.com)
```

Comment:
This line has no useful information. We should add a link after the bullet point to the actual detail page.